data/applications/aikit.yaml

```diff
@@ -0,0 +1,15 @@
+metadata:
+  name: aikit
```

Comment:
Please add an annotation for the categories the documentation can be filtered by. ie:

`  annotations:
    opendatahub.io/categories: 'AI/Machine learning,Deployment,Model serving'`

Current categories we use are: 
AI/Machine learning,
Data management,
Data visualization,
Deployment,
Getting started,
Jupyter notebook,
Model serving,
Monitory,
Python,
SciKit-Learn,
TensorFlow,
PyTorch